Zend/zend_vm_def.h

```diff
@@ -5230,7 +5230,7 @@ ZEND_VM_C_LABEL(send_array):
 			zval *op2 = GET_OP2_ZVAL_PTR(BP_VAR_R);
 			uint32_t skip = opline->extended_value;
 			uint32_t count = zend_hash_num_elements(ht);
-			zend_long len = zval_get_long(op2);
+			zend_long len = zval_get_long_ex(op2, /* is_strict */ true);
```

Comment:
This should be using zpp ... fixed in https://github.com/php/php-src/commit/27be6670b11cbe909a27d51c8e31590fc9283079.